packages/accounts/.release.sh

```diff
@@ -0,0 +1,45 @@
+# ---
+# This script could be used by release.sh in future version
+#
+# Require these env to upload source maps to Sentry:
+# - SENTRY_AUTH_TOKEN
+# - SENTRY_ORG
+# - SENTRY_PROJECT
+#
+# Require these env to upload assets to GitHub:
+# - GITHUB_TOKEN
+# or call gh auth login before uploading.
+#
+# ---
+
+PROJECT_NAME=accounts-site
+
+function build-staging-sources() {
+    npm run build:staging
+}
+
+function distribute-staging-assets() {
+    npx sentry-cli releases new "$1-staging"
+    npx sentry-cli releases files "$1-staging" upload-sourcemaps dist
+    npx sentry-cli releases finalize "$1-staging"
+
+    zip -r dist.zip dist
```

Comment:
Not quite sure whether `zip` is a pre-installed shell common on Ubuntu?

Reply:
The CI scripts should concern about software env